RDSC-3621 format epoch to date string (#1674)
redis/docs: content/integrate/redis-data-integration/data-pipelines/transform-examples/redis-expiration-example.md

In some cases, you can also set the expiration time based on a field that contains a date, datetime, or timestamp value, but it depends on the source database and the data types it supports. See the examples below for your specific source database and data type.

There are two main approaches you can use to set the expiration time based on a date, datetime, or timestamp field:

> - For values representing a passed amount of time e.g. milliseconds since epoch start, you have to convert the value to seconds since epoch and subtracting the current time in seconds since epoch from it.

Reviewer: ```suggestion
- For values representing an elapsed time since epoch start (in milliseconds, for example), you have to convert the value to seconds since epoch and then subtract the current time (also in seconds since epoch). The difference between the two is the time until expiration.
```

    ```yaml
    output:
        - uses: redis.write
          with:
            data_type: hash
            expire:
              # To set the expiration time to a date field, convert the value to
              # seconds (e.g. divide it by 1000 if the fields has milliseconds precision) 
              # and subtract the current time in seconds since epoch.
              expression: EXPIRES_TIMESTAMP / 1000 - STRFTIME('%s', 'now')
              language: sql
    ```

> - For values matching the subset of ISO 8601 supported by SQLite e.g. `2023-10-01T12:00:00`, `2023-10-01T12:00:00Z` or `2025-06-05T13:40:14.784000+02:00` you can use the `STRFTIME` function to convert the value to seconds since epoch and subtract the current time in seconds since epoch from it.

Reviewer: ```suggestion
- For values matching the subset of ISO 8601 supported by SQLite (for example, `2023-10-01T12:00:00`, `2023-10-01T12:00:00Z`, or `2025-06-05T13:40:14.784000+02:00`), you can use the `STRFTIME` function to convert the value to seconds since epoch and subtract the current time in seconds since epoch from it.
```

  ```yaml
  output:
    - uses: redis.write
      with:
        data_type: hash
        expire:
          language: sql
          expression: STRFTIME('%s', EXPIRATION_TS) - STRFTIME('%s', 'now')
  ```

> For more examples on how to manipulate date and time values, see the [Formatting date and time values]({{< relref "/integrate/redis-data-integration/data-pipelines/transform-examples/formatting-date-and-time-values/">}}) page.

Reviewer: ```suggestion
For more examples of how to manipulate date and time values, see [Formatting date and time values]({{< relref "/integrate/redis-data-integration/data-pipelines/transform-examples/formatting-date-and-time-values/">}}).
```
